pkg/rpc/response/result/invoke.go

```diff
@@ -26,7 +26,7 @@ type Invoke struct {
 
 type invokeAux struct {
 	State          string          `json:"state"`
-	GasConsumed    fixedn.Fixed8   `json:"gasconsumed,string"`
+	GasConsumed    string          `json:"gasconsumed"`
```

Comment:
But can't we just do this?
```suggestion
	GasConsumed    fixedn.Fixed8   `json:"gasconsumed"`
```

Reply:
Yes, it is simpler. Fixed.